metricbeat/module/system/diskio/_meta/data.json

```diff
@@ -1,60 +1,35 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
+    "agent": {
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "system.diskio",
+        "duration": 115000,
+        "module": "system"
+    },
     "metricset": {
-        "module": "system",
-        "name": "diskio",
-        "rtt": 115
+        "name": "diskio"
+    },
+    "service": {
+        "type": "system"
     },
     "system": {
         "diskio": {
             "io": {
-                "time": 338930
-            },
-            "iostat": {
-                "await": 0,
-                "busy": 0,
-                "queue": {
-                    "avg_size": 0
-                },
-                "read": {
-                    "per_sec": {
-                        "bytes": 0
-                    },
-                    "request": {
-                        "merges_per_sec": 0,
-                        "per_sec": 0
-                    },
-                    "await": 0
-                },
-                "request": {
-                    "avg_size": 0
-                },
-                "service_time": 0,
-                "write": {
-                    "per_sec": {
-                        "bytes": 0
-                    },
-                    "request": {
-                        "merges_per_sec": 0,
-                        "per_sec": 0
-                    },
-                    "await": 0
-                }
```

Comment:
Is the removal of this big chunk expected?

Reply:
There are automatically generated files. No need to worry about these. These are only examples for the docs.